omc3/optics_measurements/coupling.py

```diff
@@ -118,29 +120,29 @@ def calculate_coupling(
 
     LOGGER.debug("Computing complex lines from spectra")
     A01: np.ndarray = 0.5 * _get_complex_line(
-        joined[SECONDARY_AMPLITUDE_X] * exp(joined[SECONDARY_FREQUENCY_X] * PI2I), deltas_x, bpm_pairs_x
+        joined[SECONDARY_AMPLITUDE_X] * np.exp(joined[SECONDARY_FREQUENCY_X] * PI2I), deltas_x, bpm_pairs_x
```

Comment:
Appreciate that. I can't highlight it as it wasn't changed, but can we remove the line that explicitely imported these? Keeps the scope clean.